integrations/client/test_delphi_epidata.py

```diff
@@ -38,69 +39,41 @@ def setUp(self):
     # use the local instance of the Epidata API
     Epidata.BASE_URL = 'http://delphi_web_epidata/epidata/api.php'
 
+    # use the local instance of the epidata database
+    secrets.db.host = 'delphi_database_epidata'
+    secrets.db.epi = ('user', 'pass')
+
```

Comment:
Why is this needed?

Reply:
The meta update test in this file fails without this addition *unless* some prior test happened to set it already. i.e. this makes it deterministic.

For example, this command fails:
```
docker run --rm --network delphi-net delphi_python python3 -m undefx.py3tester.py3tester --color repos/delphi/delphi-epidata/integrations/client
```

Whereas this command passes:
```
docker run --rm --network delphi-net delphi_python python3 -m undefx.py3tester.py3tester --color repos/delphi/delphi-epidata/integrations
```

The reason the latter passes is because some test in `integrations/acquisition` sets the database host/user/pass _before_ the meta update test in `integrations/client`.